### PR TITLE
Sort classes on format

### PR DESCRIPTION
**Pull request by bartosztrusinski**
Add plugin to automatically sort Tailwind classes on format 🍃

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "jsxSingleQuote": true,
-  "bracketSameLine": true
+  "bracketSameLine": true,
```
Comment (nikkehtine) [inline .prettierrc]: I think setting this to false would result in slightly better visibility, let me know what everyone else thinks about this @Azzirr @bartosztrusinski

**Review by nikkehtine — APPROVED** [review]

**Review by mproznydev — APPROVED** [review]

File: .prettierrc (hunk 1 above)
Comment (Azzirr) [reply .prettierrc]: Honestly I don't know how it works, but i found that against this (on true):
  return { name: "John", age: 30 };

we will have this (on false):
  return { 
    name: "John", 
    age: 30 
  };
  
  If yes - I agree with @nikkehtine, better visability is on false

**Review by Azzirr — APPROVED** [review]

File: .prettierrc (hunk 1 above)
Comment (bartosztrusinski) [reply .prettierrc]: image

this is the rule, closing > on separate line looks bad and bloats file but i can set it to false if everyone wants that

**Review by mproznydev — APPROVED** [review]

File: .prettierrc (hunk 1 above)
Comment (bartosztrusinski) [reply .prettierrc]: okay, set this rule to `false`, merging

**Pull request merged**